numpy/ma/extras.py

```diff
@@ -48,7 +48,7 @@
 from numpy import ndarray, array as nxarray
 import numpy.core.umath as umath
 from numpy.lib.index_tricks import AxisConcatenator
-from numpy.linalg import lstsq
+from numpy.lib.function_base import _CORRCOEF_MSG_FMT, _DefaultArg
```

Comment:
Since its use is entirely local, it might be better to just define it in each module in which it is used. It is going to be temporary, at least in the numpy sense of temporary. Otherwise, I might suggest `numpy/__init__.py`, which already has a couple of global classes. In the latter case a more suggestive name might help.